package.json

```diff
@@ -9,35 +9,41 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "type-check": "tsc",
+    "check": "npm run lint && npm run prettier && npm run typecheck",
```

Comment:
I wasn't sure if I need to run the prettier here explicitly or if it's ran automatically during eslint run @oleg-semyonov ? 🤔

Reply:
It is actually run with eslint, so we can skip it here